lib/widgets/message_list.dart

```diff
@@ -392,12 +393,18 @@ class MarkAsReadWidget extends StatelessWidget {
           // TODO(#368): this should pull from stream color
           color: Colors.transparent,
           child: Padding(
-            padding: const EdgeInsets.all(10),
+            // vertical padding adjusted for tap target height (48px) of containing button
```

Comment:
nit:
```suggestion
            // vertical padding adjusted for tap target height (48px) of button
```

From the perspective of this `Padding`, the button isn't containing, but rather contained — it's the child of the padding.